modules/oe_content_call_proposals/config/optional/language.content_settings.node.oe_call_proposals.yml

```diff
@@ -0,0 +1,10 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - node.type.oe_call_proposals
+id: node.oe_call_proposals
```

Comment:
lets specify the module and third party settings like here https://github.com/openeuropa/oe_content/blob/master/modules/oe_content_call_tenders/config/optional/language.content_settings.node.oe_call_tenders.yml#L6

Reply:
This is done.